vitrina/datasets/forms.py

```diff
@@ -102,27 +117,35 @@ class DatasetForm(TranslatableModelForm, TranslatableModelFormMixin):
                 message="Kodinis pavadinimas turi būti sudarytas iš mažųjų raidžių ir (arba) gali turėti pasvirųjų brūkšnių",
             )
         ],
+        help_text=_(
+            "Kodinis pavadinimas yra neprivalomas (jei nepateiktas, sugeneruojamas automatiškai iš pavadinimo) duomenų rinkinio pavadinimas, sudarytas iš mažųjų raidžių ir (arba) gali turėti pasvirųjų brūkšnių."
+        ),
     )
 
     contact = forms.ChoiceField(
-        label=_("Kontaktinis asmuo ar organizacija"), required=False
+        label=_("Kontaktinis asmuo ar organizacija"),
+        required=False,
+        help_text=_("Nurodomas asmuo arba organizacija atsakinga už duomenų rinkinio turinį."),
```

Comment:
```suggestion
        help_text=_("Nurodomas asmuo arba organizacija atsakinga už duomenų rinkinio turinį. Šiuo adresu bus nukreipiami visi paklausimai dėl duomenų rinkinio."),
```

Nors ir parašyta „dėl duomenų rinkinio“, bet tai priklauso nuo pasirinkti ištekliaus tipo.